utils/s2n_safety.c

```diff
@@ -90,12 +90,21 @@ int s2n_constant_time_copy_or_dont(uint8_t * dest, const uint8_t * src, uint32_t
     S2N_PUBLIC_INPUT(src);
     S2N_PUBLIC_INPUT(len);
```

Comment:
Don't we need to check for nullness in `dest`, `src`? Actually, we need to check for `S2N_MEM_IS_READABLE` and `S2N_MEM_IS_WRITABLE` (check out `s2n_errno.h`).

Reply:
What about this one?